README.md

```diff
@@ -2,74 +2,85 @@
 
 ## Description
 
-This tool is a first draft of connecting an LMS interface to a Graylog server to create a dataset 
-of learning logs.
+This tool is dedicated to generate learning events (_aka_ tracking logs) from the Edx platform to feed dashboard development testing datasets.
+It is intended to be used locally.
 
-It is intended to be used locally. 
-Currently, it is using [OpenEdx](https://github.com/openfun/openedx-docker)'s LMS service 
-(with the fun-flavored dogwood's release).
+It has been duplicated from [OpenEdx](https://github.com/openfun/openedx-docker) fun-flavored dogwood's release
+and integrates a Graylog server connected to the LMS service.
 
-## Getting started
+## Usage
 
-- Clone the current repository.
+### Launch Edx platform
 
-- When using the tool for the first time, you need to run migrations. Run the following command:
+1. Run the following command to launch the Edx platform:
 
 ```
-make migrate
+make bootstrap
 ```
 
-The tool is now ready for use ! :rocket:
+> Note: as the command includes migration, it may take some tome to complete (several minutes). Go grab a coffee, a tea or whatever pleases you!
```

Comment:
My bad :sweat_smile: 

```suggestion
> Note: as the command includes migration, it may take some time to complete (several minutes). Go grab a coffee, a tea or whatever pleases you!
```